index.html

```diff
@@ -2273,180 +2273,94 @@ <h3>Processing Errors</h3>
-verifiable credentials or other data-integrity-protected documents, from
-any issuer and using any contexts, might need to be able to load externally
-linked resources, such as JSON-LD context files, in production settings. This is
-expected to increase user choice, scalability, and decentralized upgrades in the
-ecosystem over time. Authors of such applications are advised to read the
-security and privacy sections of this document for further considerations.
-      </p>
-
-      <p>
-For further information regarding processing of JSON-LD contexts and
-vocabularies, see <a data-cite="?VC-DATA-MODEL-2.0#base-context">Verifiable
-Credentials v2.0: Base Context</a> and <a data-cite="?VC-DATA-MODEL-2.0#vocabularies">
-Verifiable Credentials v2.0: Vocabularies</a>.
+        <strong>SHA2-256 Digest:</strong> <code><span class="vc-hash"
+data-hash-url="https://w3c.github.io/vc-data-integrity/vocab/security/vocabulary.html"
+data-hash-format="openssl dgst -sha256" /></code>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+<p>
+  It is possible to confirm the cryptographic digests listed above by running
+  a command like the following (replacing `&lt;MEDIA_TYPE>` and `&lt;DOCUMENT_URL>`
+  with the appropriate values) through a modern UNIX-like OS command line interface:
+  `curl -sL -H "Accept: &lt;MEDIA_TYPE>" &lt;DOCUMENT_URL> | openssl dgst -sha256`
+</p>
+
+
+      <p class="note">
+Applications or specifications that are based on JSON-LD processing should define mappings to the 
+vocabulary URLs using JSON-LD contexts. For example, for the terms defined in this
+specification, such mappings are part of the `https://w3id.org/security/data-integrity/v2` context, 
+defined by the [[[?VC-DATA-INTEGRITY]]] specification, or the `https://www.w3.org/ns/did/v1` context, 
+defined by the [[[?DID-CORE]]] specification.
       </p>
 
-      <section>
-        <h3>Context Injection</h3>
-
-        <p>
-The `@context` property is used to ensure that implementations are using the
-same semantics when terms in this specification are processed. For example, this
-can be important when properties like `type` are processed and its value, such
-as `DataIntegrityProof`, are used.
-        </p>
-
-        <p>
-If an `@context` property is not provided in a document that is being secured or
-verified, or the Data Integrity terms used in the document are not mapped by
-existing values in the `@context` property, implementations MUST inject or add
-an `@context` property with a value of
-`https://w3id.org/security/data-integrity/v2`.
-        </p>
-
-        <p>
-Context injection is expected to be unnecessary sometimes, such as when the Verifiable
-Credential Data Model v2.0 context (`https://www.w3.org/ns/credentials/v2`)
-exists as a value in the `@context` property, as that context maps all of the
-necessary Data Integrity terms that were previously mapped by
-`https://w3id.org/security/data-integrity/v2`.
-        </p>
-      </section>
```

Comment:
I think removing this is fine because it's not about the controller document. However, I think this section should say that it is optional to inject the multikey context or the jsonwebkey context when resolving keys -- and then provide the hashes for those and the "already resolved" advice.

Reply:
There is also a base context for the controller document, which includes the multikey and jsonwebkey contexts. This isn't as cut and dried as it might seem. Injecting `https://w3id.org/security/data-integrity/v2` for JSON-LD is clearly wrong... but we should say something about injecting `https://www.w3.org/ns/controller/v1`.

Reply:
@msporny, as I said in https://github.com/w3c/controller-document/pull/43#issuecomment-2324716579, I have not heard about a controller context file so far. And I am not convinced about its necessity, to be honest. B.t.w., neither DI or DID would use it; after all, both have already contexts spelling out the controller terms in their own things.